frontend/src/bundles/employers/types/employers-data-dto.ts

```diff
@@ -0,0 +1,11 @@
+type EmployerDataDto = {
+    companyWebsite: string;
+    fullName: string;
+    companyLogo: { url: string };
```

Comment:
I think it should be like general File type like { id: string, url: string ... } or something like that

Reply:
Fixed